dev-cli/src/commands/spawn.js

```diff
@@ -4,22 +4,30 @@ import { Command } from '../deps.js'
 import { tagsArg, walletArgs } from '../utils.js'
 import { VERSION } from '../versions.js'
 
-function sourceArgs (src) {
+function moduleArgs (src) {
   return [
     '-e',
     `MODULE_TX=${src}`
   ]
 }
 
+function schedulerArgs (src) {
+  return [
+    '-e',
+    `SCHEDULER_ADDRESS=${src}`
```

Comment:
I don't see where this is used in `ao-spawn.js`. I would suspect that it would be mapped to `scheduler` when calling `uploadAoProcess`, similar to `WALLET_PATH` and `MODULE_TX`